sydent/db/hashing_metadata.sql

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2019 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/* 
+ * hashing_metadata contains information needed for the identity server to carry
+ * out tasks related to hashing. Salts and peppers etc. should go here.
+ */
+CREATE TABLE IF NOT EXISTS hashing_metadata (
+    lookup_pepper varchar(256)
```

Comment:
Adding a maximum size to columns generally only ends in woe, ime, so I'd not bother.